syne_tune/optimizer/schedulers/hyperband.py

```diff
@@ -373,6 +376,7 @@ def __init__(self, config_space, **kwargs):
         self.terminator = HyperbandBracketManager(
             scheduler_type, self._resource_attr, self.metric, self.mode,
             self.max_t, rung_levels, brackets, rung_system_per_bracket,
+            num_points_to_evaluate=len(kwargs.get('points_to_evaluate', list())),
```

Comment:
Are you suggesting that we need to support the case that someone wants to provide initial configs which aren't be used as threshold candidates for RUSH? I thought it might be very convenient to simply pass custom points_to_eval instead of using any metadata.